src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1889,7 +1889,7 @@ protected override LocalState ReachableBottomState()
 
         private void EnterParameters()
         {
-            if (!(_symbol is MethodSymbol methodSymbol))
+            if (!(CurrentSymbol is MethodSymbol methodSymbol))
```

Comment:
>CurrentSymbol [](start = 18, length = 13)

There are other references to `_symbol` that are not updated. Are those other references correct?

Reply:
I made a pass over existing usages of `_symbol` and changed the ones that seemed wrong.